.github/workflows/package.yml

```diff
@@ -46,10 +46,9 @@ jobs:
       - name: Push Release
         uses: docker/build-push-action@v5
         with:
-          push: ${{ startsWith(github.ref, 'refs/tags/')  || github.ref == 'refs/heads/next' }}
+          push: ${{ startsWith(github.ref, 'refs/tags/')  || github.ref == 'refs/heads/next' || github.ref == 'refs/heads/main' }}
```

Comment:
What do you think about extracting this into an environment variable within the workflow?

It would follow the [DRY principle](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself), and there is less risk of making a mistake as this condition is defined in one location.

Reply:
It does not duplicate, so don't need to create variable